modules/apps/app-builder/app-builder-test/src/testFunctional/paths/FormViewBuilder.path

```diff
@@ -365,6 +365,18 @@
 	<td></td>
 </tr>
 
+<tr>
```

Comment:
Hi @calvinctvo , these selectors are not needed since you are not dealing with rules and the Save action can be replaced by the macro available at FormViewBuilder macro.